test/core/try_catch.wast

```diff
@@ -0,0 +1,206 @@
+;; Test try-catch blocks.
+
+(module
+  (event $e0 (export "e0"))
+  (func (export "throw") (throw $e0))
+)
+
+(register "test")
+
+(module
+  (event $imported-e0 (import "test" "e0"))
+  (func $imported-throw (import "test" "throw"))
+  (event $e0)
```

Comment:
Assuming that we are changing this to `(tag ...)` (not necessarily in this PR), do we need a text syntax to represent the attribute (exception)? Currently we are tentatively using 
```wasm
(event $e0 (attr 0))
```
in Binaryen, but not sure if this is a good one. (I uploaded a PR to change this in Binaryen to `(tag ..)` in WebAssembly/binaryen#3937, but this `attr` thing is a separate thing anyway)

Reply:
As argued elsewhere, I don't think it is necessary to manifest the (so far vacuous) attribute in the text format for the time being. It's not even necessary to put it in the semantics. All we need right now is a reserved zero byte in the binary format, until we know better what possible distinctions are needed in the future.

Reply:
So if we eventually need a different tag kind and go with @aheejin's syntax, exception tags would be a special case as shown below?
```
(tag)          ;; an exception tag
(tag (attr 0)) ;; an exception tag
(tag (attr 1)) ;; some other tag
```
I am fine with that, just want to clarify.

Reply:
Sounds good. I'll remove the `attr` attribute from the Binaryen text format then.